### PR TITLE
DEVX-2241: Update Elasticsearch and Kibana

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2241

_What behavior does this PR change, and why?_

* Current ES v5.6 --> update to ES v7.10.
* Work borrowed from https://github.com/robcowart/cp-demo/tree/nextgen_elastic
* @jeanlouisboudart @waliaabhishek : assess impact to `jmx-monitoring-stacks` for 6.1.x

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation
- [x] Run cp-demo

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

- [ ] Run cp-demo
- [ ] jmx-monitoring-stacks (cc: @waliaabhishek @jeanlouisboudart )

### CODE REVIEW

**Review by mikebin — APPROVED** [review]
LGTM

**Review by rspurgeon — APPROVED** [review]
LGTM   Tested locally.  Nice changes

File: Dockerfile [hunk 1]
```diff
@@ -28,7 +28,7 @@ RUN confluent-hub install --no-prompt cjmatta/kafka-connect-sse:latest
 RUN confluent-hub install --no-prompt jcustenborder/kafka-connect-json-schema:latest
 
 # Install Elasticsearch connector
-RUN confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:10.0.2
+RUN confluent-hub install --no-prompt confluentinc/kafka-connect-elasticsearch:latest
```
Comment (javabrett) [inline Dockerfile:31]: Checking - do we intend to re-pin this later, or let if float?  If re-pin, might be worth adding a TODO comment.

File: scripts/dashboard/configure_kibana_dashboard.sh [hunk 2]
```diff
@@ -1,10 +1,5 @@
...
-curl -X POST -H "kbn-version: 5.5.2" -H "Content-Type: application/json;charset=UTF-8" -d '{"value":"META.DT"}' http://localhost:5601/api/kibana/settings/timelion:es.timefield
-curl -X POST -H "kbn-version: 5.5.2" -H "Content-Type: application/json;charset=UTF-8" -d '{"value":"wikipediabot"}' http://localhost:5601/api/kibana/settings/defaultIndex
-curl -X PUT -H "kbn-version: 5.5.2" -d '{"title":"en_wikipedia_gt_1","notExpandable":true}' http://localhost:5601/es_admin/.kibana/index-pattern/en_wikipedia_gt_1/_create 
-curl --silent --output /dev/null -X POST -H 'kbn-xsrf:true' -H 'Content-type:application/json' -d @scripts/dashboard/kibana_dash.json http://localhost:5601/api/kibana/dashboards/import
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-#curl -X GET http://localhost:5601/api/kibana/dashboards/export?dashboard=Wikipedia > /tmp/dash.json
+curl -XPOST "http://localhost:5601/api/saved_objects/_import?overwrite=true" -H "kbn-xsrf: true" -H "securitytenant: global" --form file=@${DIR}/cp-demo.ndjson
```
Comment (javabrett) [inline scripts/dashboard/configure_kibana_dashboard.sh:5]: I'm sure this works, but I'm used to seeing a space here and for readability `-X POST` - is nospace better?

**Review by javabrett — APPROVED** [review]
Outstanding!  I am embarrassed to confess I have a _very_ stale local branch `elasticsearch-kibana-upgrade` that is now heading to `/dev/null`.  Thanks @robcowart for the contributions.

File: Dockerfile (hunk 1 above)
Comment (ybyzek) [reply Dockerfile]: @javabrett -- PR is now updated to pin to 11.0.0 (cc: @rspurgeon )

File: scripts/dashboard/configure_kibana_dashboard.sh (hunk 2 above)
Comment (ybyzek) [reply scripts/dashboard/configure_kibana_dashboard.sh]: PR is updated with space

**Comment by rspurgeon** [comment]
Still LGTM

**Pull request merged**